larray/viewer/view.py

```diff
@@ -1316,8 +1316,9 @@ def setup_menu_bar(self):
                                           triggered=self.close))
 
         help_menu = menu_bar.addMenu('&Help')
-        help_menu.addAction(create_action(self, _('Online documentation'), shortcut="Ctrl+H",
-                                          triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('Online &Documentation'), shortcut="Ctrl+H", triggered=self.open_documentation))
+        help_menu.addAction(create_action(self, _('&Getting Started'), triggered=self.open_tutorial))
+        help_menu.addAction(create_action(self, _('Objects and &Functions'), triggered=self.open_api_documentation))
```

Comment:
Our main users don't know what API means. That's why I didn't mentioned it. But OK, I could add in ( ).